Topics/Teamwork.md

```diff
@@ -12,3 +12,25 @@ An important first step towards conflict resolution is learning how to avoid the
 [Resolving Conflicts](https://ca.indeed.com/career-advice/career-development/conflict-resolution-at-work)
 
 It is not always possible to avoid conflict. In work environment when people are exchanging ideas and working towards goals, a difference of opinion is inevitable. However, if handled properly, these conflicts can be resolved while keeping the negative impacts at a minimum, and even help create a better solution for the project at hand. This article covers ways that the parties in a conflict can help each other understand their own points of view, while keeping discourse respectful. It emphasizes ways that team members can help each other work towards a common goal.
+
+
+
+
+
+## **Roles in a Software Team **
```

Comment:
```suggestion
## Roles in a Software Team
```

The usage of `**` (bold) won't work here since the line is a title (`##`). So when viewing the rendered Markdown file, I end up seeing `**Roles in a Software Team **`.